evm_loader/program/src/account_storage.rs

```diff
@@ -0,0 +1,253 @@
+use crate::solana_backend::AccountStorage;
+use evm::{
+    backend::{Basic, Backend, Apply, Log},
+    CreateScheme, Capture, Transfer, ExitReason
+};
+use core::convert::Infallible;
+use primitive_types::{H160, H256, U256};
+use sha3::{Digest, Keccak256};
+use solana_sdk::{
+    account_info::{next_account_info, AccountInfo},
+    pubkey::Pubkey,
+    program_error::ProgramError,
+    sysvar::{clock::Clock, Sysvar},
+    instruction::{Instruction, AccountMeta},
+};
+use std::{
+    cell::RefCell,
+};
+
+use crate::solidity_account::SolidityAccount;
+use crate::account_data::AccountData;
+use solana_sdk::program::invoke;
+use solana_sdk::program::invoke_signed;
+use std::convert::TryInto;
+use arrayref::{array_ref, array_refs, array_mut_ref, mut_array_refs};
+
+fn keccak256_digest(data: &[u8]) -> H256 {
+    H256::from_slice(Keccak256::digest(&data).as_slice())
+}
+
+fn u256_to_h256(value: U256) -> H256 {
+    let mut v = vec![0u8; 32];
+    value.to_big_endian(&mut v);
+    H256::from_slice(&v)
+}
```

Comment:
it is better to move these functions to the `evm_loader/program/src/utils.rs` as they are used in different parts of the program, but do not relate directly to AccountStorage.